contrib/userscript/unnoticed.user.js

```diff
@@ -0,0 +1,201 @@
+// ==UserScript==
+// @name         Unnoticed Leaderboards
+// @namespace    https://github.com/christopher-dG/unnoticed
+// @version      0.1
+// @description  Display unranked leaderboard entries gathered by Unnoticed on their respective beatmap pages
+// @author       Node
+// @match        https://osu.ppy.sh/*/*
+// @include      https://p9bztcmks6.execute-api.us-east-1.amazonaws.com/*
+// @grant        GM_xmlhttpRequest
+// @require      https://code.jquery.com/jquery-3.2.1.min.js
+// @require      http://momentjs.com/downloads/moment.min.js
+// @connect      p9bztcmks6.execute-api.us-east-1.amazonaws.com
+// @run-at document-end
+// ==/UserScript==
+
+(function(){
+    'use strict';
+
+    var mods_enum = {
+        ''   : 0,
+        'NF' : 1,
+        'EZ' : 2,
+        ''   : 4,
+        'HD' : 8,
+        'HR' : 16,
+        'SD' : 32,
+        'DT' : 64,
+        'RX' : 128,
+        'HT' : 256,
+        'NC' : 512,
+        'FL' : 1024,
+        'AT' : 2048,
+        'SO' : 4096,
+        'AP' : 8192,
+        'PF' :16384 
+    };
+
+    function accuracy(c300, c100, c50, cmiss){
```

Comment:
This will need to be updated for multiple game modes, you can see an example [here](https://github.com/christopher-dG/unnoticed/blob/master/discord/bot.rb#L155-L175).

Reply:
I will add additional parameters once other modes are supported.